app/javascript/mastodon/features/terms_of_service/index.jsx

```diff
@@ -115,8 +114,8 @@ class TermsOfService extends React.PureComponent {
               Mozilla.social account. However, we may retain such data to the
               extent necessary in order to enforce our policies or comply with
               the law, as described in the Mozilla.social Privacy Notice. This
-              will not delete your Firefox account, which you may delete
-              separately on your Firefox account <a href='https://accounts.firefox.com/'>settings page</a>.</p>
+              will not delete your Mozilla account, which you may delete
+              separately on your Mozilla account <a href='https://accounts.firefox.com/'>settings page</a>.</p>
 
```

Comment:
It's not entirely clear how to use just "account" rather than "Mozilla account" here, as the copy needs to differentiate between the user's MozSocial and FxA/Mozilla account. I'd love some help with this, or a 👍 to leave as is.